DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,204 @@
+            cy.get("#lastName").type(data.lastName)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type("+8493300884")
+
+            cy.get("#submit").click({ force: true })
+
+            cy.get("#userNumber").should('have.css', 'border-color', 'rgb(220, 53, 69)');
+
+        })
+    })
+
+    it('cannot submit with all empty', () => {
+
+        cy.get("#submit").click({ force: true })
+
+        cy.get("#firstName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        cy.get("#lastName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        cy.get("label[for='gender-radio-1']").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        cy.get("#userNumber").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+    })
+
+    it('cannot submit with firstName and LastName have specChar ', () => {
+
+        cy.fixture('testData').then((data) => {
+            cy.get("#firstName").type(data.firstName + '@#')
+            cy.get("#lastName").type(data.lastName + '$%^')
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(data.userNumber)
+
+            cy.get("#submit").click({ force: true })
+
+            cy.get("#firstName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+            cy.get("#lastName").should('have.css', 'border-color', 'rgb(220, 53, 69)')
+        })
+    })
+
+    it('cannot submit with invalid mobile ', () => {
+
+        testData.invalidNumbers.forEach((mobile, index) => {
+            cy.get("#firstName").type(testData.firstName)
+            cy.get("#lastName").type(testData.lastName)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(mobile)
+
+            cy.get("#submit").click({ force: true })
+
+            cy.get("#userNumber").should('have.css', 'border-color', 'rgb(220, 53, 69)');
+        })
+    })
+
+    it('cannot submit with an invalid email', () => {
+        testData.invalidEmail.forEach((email, index) => {
+
+            cy.get("#firstName").type(testData.firstName)
+            cy.get("#lastName").type(testData.lastName)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(testData.userNumber)
+            cy.get('#userEmail').type(email)
+
+            cy.get("#submit").click({ force: true })
+
+            cy.get("#userEmail").should('have.css', 'border-color', 'rgb(220, 53, 69)');
+        })
+    })
+
+    it('cannot submit with DOB with later than today', () => {
+
+        cy.fixture('testData').then((data) => {
+            cy.get("#firstName").type(data.firstName)
+            cy.get("#lastName").type(data.lastName)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(data.userNumber)
+            cy.get('#dateOfBirthInput').click()
+            cy.get('.react-datepicker__month-select').select('January')
```

Comment:
Just as I comment above, try to use `Date()`. If you work in a project that active in a long time, maybe for years, your test data will be wrong someday in the future, `Date()` can help you get the current day and add one day or one year